### DISCUSSION TITLE
Change Start Button image

### DISCUSSION THREAD

**Discussion by Skyyblaze** [ramensoftware/windhawk-mods#370] [Mod ideas]
Would the Windhawk system be able to change the icon of the Start Button to a custom one?

**Comment by m417z** [comment]
Currently there's no such mod. There's an open issue about it that you can follow: https://github.com/ramensoftware/windhawk-mods/issues/175

**Comment by PushoDev** [comment]
Hi everyone! I found a way to change the Start Button icon without third-party apps like Start11, using only the Windows 11 Taskbar Styler mod in Windhawk.
Here are the steps:

- Open Windows 11 Taskbar Styler and choose your preferred theme (e.g., SimplyTransparent).
- Go to the Advanced tab.
- In the JSON configuration, you need to add two specific control styles: one to hide the original animated icon and another to set your custom image as the background.

Important: Make sure to change the ImageSource path to your actual image location (e.g., C:\\path\\to\\your\\icon.png).
Here is the complete JSON structure (using "SimplyTransparent" as an example):

```
{
  "theme": "SimplyTransparent",
  "controlStyles[0].target": "Taskbar.ExperienceToggleButton#LaunchListButton[AutomationProperties.AutomationId=StartButton] > Taskbar.TaskListButtonPanel > Border#BackgroundElement",
  "controlStyles[0].styles[0]": "Background:=<ImageBrush Stretch=\"Uniform\" ImageSource=\"C:\\tools\\arch_inicio.png\"/>",
  "controlStyles[1].target": "Taskbar.ExperienceToggleButton#LaunchListButton[AutomationProperties.AutomationId=StartButton] > Taskbar.TaskListButtonPanel > Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer#Icon",
  "controlStyles[1].styles[0]": "Visibility=Collapsed",
  "styleConstants[0]": "",
  "resourceVariables[0].variableKey": "",
  "resourceVariables[0].value": "",
  "themeResourceVariables[0]": "",
  "xamlDiagnosticsHandling": "alert"
}
```

Why this works:

- controlStyles[0] targets the background of the Start button and replaces it with an ImageBrush.
- controlStyles[1] targets the default animated icon (AnimatedVisualPlayer) and sets it to Collapsed so it doesn't overlap with your new icon.

If this solution helps you, please mark it as the answer to help others! 🚀
<img width="1919" height="1079" alt="Screenshot 2026-03-29 165057" src="https://github.com/user-attachments/assets/bce03ed9-810b-4739-9a99-196eb9e7cb74" />
